x-pack/plugins/security_solution/server/lib/detection_engine/signals/source_fields_merging/README.md

```diff
@@ -0,0 +1,389 @@
+Set of utilities for merging between `_source` and `fields` are within this folder as well as strategies for merging between these two.
+
+See `strategies` for the strategies for merging between `_source` and `fields`. See the `utils` folder for the different utilities
+which the strategies utilize for help in building their merged documents.
+
+If we run into problems such as ambiguities, uncertainties, or data type contradictions then we will prefer the value within
```

Comment:
From discussion, this depends on which algorithm, correct? The default one will use `doc._source` -- worth categorizing each algorithm and their corresponding behavior in this readme?

Reply:
This is correct. I will update these docs here.